docs/api/main-config-babel-default.md

```diff
@@ -26,7 +26,9 @@ To adjust your Storybook's Babel setup directly—not via an addon—use [`babel
 
 ## `Babel.Config`
 
-See [Babel docs](https://babeljs.io/docs/options).
+The options provided by [Babel](https://babeljs.io/) are only applicable if you've enabled the [`@storybook/addon-webpack5-compiler-babel`](https://storybook.js.org/addons/@storybook/addon-webpack5-compiler-babel) addon.
```

Comment:
Why the change in link URL? The previous one was more helpful, imo.

Reply:
This was accidentally introduced, I forgot to update the link accordingly, but it should be all fixed now